RecoEgamma/PhotonIdentification/test/runtests.sh

```diff
@@ -1,10 +1,10 @@
 function die { echo $1: status $2 ;  exit $2; }
 
 ids_to_test=(
-"RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_25ns_V1_cff"
```

Comment:
If I'm not mistaken, this is still a part of miniAOD and should be included in the tests as long as it's in the standard configurations. This comment applies to all IDs in this and the electron runtests.sh

Reply:
Dear @slava77 , yes it is in the miniAOD now, but as per my understanding we have said above, that we will keep in the miniaod_tools sequence only 2016 and 2017 IDs. This si why I removed the 2015 ID from this script (as well for photons.). Do you want me to keep these 2015 IDs in the runtests.sh as well?
Thanks
Livia

Reply:
Hi Livia,
please keep changes consistently in one PR.
IIUC, the plan is only to cleanup the standard configuration in miniaod_tools.
If that's the case, the runtests.sh have to still include all existing IDs even in the upcoming PR.
If, on the other hand, your intention is to also delete all code that provides these older IDs, then the cleanup in runtests.sh is OK, but still only together with the full cleanup in the code.

Reply:
Dear @slava77 , 2015 Ids are back in the PR now.